dbehavior:
  role:
   striker:
    start_pos: [-75, -300, 90]
    init_pos: [-240, 0, 0]
    kickoff_pos: [-200, 0, 0]
    patrol_pos_a: [150, -130, 0]
    patrol_pos_b: [300, -130, 180]
    team_radius: 100
   supporter:
    start_pos: [-75, -300, 90]
    init_pos: [-150, 150, -45]
    kickoff_pos: [-75, 150, 0]
    patrol_pos_a: [150, 130, 0]
    patrol_pos_b: [300, 130, 180]
    team_radius: 150
   defender:
    start_pos: [-300, -300, 90]
    init_pos: [-250, 0, 90]
    patrol_pos_a: [-200, -200, 90]
    patrol_pos_b: [-200, 200, -90]
    team_radius: 200
    defend_radius: 100
   goalkeeper:
    start_pos: [-300, -300, 90]
    init_pos: [-400, 0, 0]
    patrol_pos_a: [-400, -130, 90]
    patrol_pos_b: [-400, 130, -90]
    team_radius: 200
   penaltykicker:
    start_pos: [-300, 0, 180]
    init_pos: [-300, 0, 180]
    patrol_pos_a: [0, 0, 0]
    patrol_pos_b: [0, 0, 0]
    team_radius: 200
   others:
    start_pos: [-450, 0, 0]
    init_pos: [-450, 0, 0]
    patrol_pos_a: [-150, 150, 0]
    patrol_pos_b: [-150, -150, 180]
    team_radius: 200
